models/ed/R/model2netcdf.ED2.R

```diff
@@ -21,25 +22,155 @@
 ##'
 ##' @author Michael Dietze, Shawn Serbin, Rob Kooper, Toni Viskari, Istem Fer
 ## modified M. Dietze 07/08/12 modified S. Serbin 05/06/13
-model2netcdf.ED2 <- function(outdir, sitelat, sitelon, start_date, end_date) {
-
-  flist <- dir(outdir, "-T-")
-  if (length(flist) == 0) {
-    print(paste("*** WARNING: No tower output for :", outdir))
+model2netcdf.ED2 <- function(outdir, sitelat, sitelon, start_date, end_date, pft_names = NULL, dbh_breaks = 0) {
```

Comment:
Added two new arguments but didn't update the roxygen tags to define what those variables are.